server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -109,6 +116,22 @@ function selectProtocolVersion(connectVersions: string[]): string | undefined {
 			}
 		}
 	}
+	return undefined;
+}
+
+function checkVersion(versions: string[]): [string[], string] {
+	// Iterate over the version ranges provided by the client and select the best one that works
+	const connectVersions = versions ? versions : ["^0.1.0"];
+	const version = selectProtocolVersion(connectVersions);
+	if (!version) {
+		throw new NetworkError(
+			400,
+			`Unsupported client protocol. Server: ${protocolVersions}. Client: ${JSON.stringify(
```

Comment:
`protocolVersions` and `connectVersions` are both string arrays so either both of them need `JSON.stringify()`, or neither does, I think?

Reply:
Nice catch, this was from old code, all I did was just extract to a new method. Addressed